test/unit/math/prim/fun/zeros_row_vector_test.cpp

```diff
@@ -0,0 +1,20 @@
+#include <stan/math/prim.hpp>
+#include <test/unit/math/prim/fun/expect_matrix_eq.hpp>
+#include <gtest/gtest.h>
+
+TEST(MathFunctions, zeros_row_vector) {
+  using Eigen::RowVectorXd;
+  using stan::math::zeros_row_vector;
+
+  RowVectorXd u0 = zeros_row_vector(0);
+  EXPECT_EQ(0, u0.size());
+
+  for (int K = 1; K < 5; K++) {
+    Eigen::RowVectorXd v = Eigen::RowVectorXd::Zero(K);
```

Comment:
Stick to either the `using` with no qualification or qualification.  As is, there's one `Eigen::RowVectorXd` and one just `RowVetorXd`.